### PR TITLE
Public PHActsSiliconSeeding setters for cluster cuts

### PR DESCRIPTION
**Pull request by hrjheng**
[comment]: <> (Please tell us something about this pull request)

## Types of changes
[comment]: <> ( What types of changes does your code introduce? Put an `x` in all the boxes that apply: )
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work for users)
- [ ] Requiring change in macros repository (Please provide links to the macros pull request in the last section)
- [x] I am a member of [GitHub organization of sPHENIX Collaboration](https://github.com/orgs/sPHENIX-Collaboration/people), EIC, or ECCE (contact Chris Pinkenburg to join)

## What kind of change does this PR introduce? (Bug fix, feature, ...)

Adds public PHActsSiliconSeeding setters to configure a cluster minimum ADC and maximum phi-size cut. Both cuts default to disabled (-1). 
(The large number of changes come from the clang-format that I accidentally triggered.)

[comment]: <> ( What does this PR do? Linking to talk in software meeting encouraged )

## TODOs (if applicable)

[comment]: <> ( In case this is a draft PR, e.g. for running checks using Jenkins, please make the pull request as a draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/  )

## Motivation / context

This change supports systematic studies of INTT cluster selection in `PHActsSiliconSeeding`. Both cuts are disabled by default with a value of `-1`.

## Key changes

- Added public setters for minimum INTT cluster ADC and maximum phi-size.
- Applied the cuts during INTT matching, layer iteration, and space-point construction.
- Added high-verbosity output for the configured cut values.
- Applied clang-format updates.

## Potential risk areas

- Enabling either cut can change reconstruction results.
- No I/O format or thread-safety changes are indicated.
- The checks add small per-cluster processing overhead.
- Test results are unavailable.

## Possible future improvements

- Add unit tests for cut boundaries.
- Add integration tests with cuts disabled and enabled.
- Document cut semantics and recommended values.

AI-generated summaries can contain mistakes. Contributors should verify the implementation and test results before merging.

### CODE REVIEW

File: offline/packages/trackreco/PHActsSiliconSeeding.cc [hunk 1]
```diff
@@ -824,45 +772,59 @@ std::vector<short int> PHActsSiliconSeeding::getInttCrossings(TrackSeed& si_trac
...
-std::vector<TrkrDefs::cluskey> PHActsSiliconSeeding::findMatches(
-    std::vector<Acts::Vector3>& clusters,
-    std::vector<TrkrDefs::cluskey>& keys,
-    TrackSeed& seed)
+// function for INTT cluster cuts
+bool PHActsSiliconSeeding::passesInttClusterCuts(const TrkrCluster &cluster) const
 {
-  auto fitpars = m_zeroField ? TrackFitUtils::fitClustersZeroField(clusters, keys, true)
-                             : TrackFitUtils::fitClusters(clusters, keys, true);
+  if (m_inttClusterAdcMin > 0 && cluster.getAdc() <= static_cast<unsigned int>(m_inttClusterAdcMin))
```
[the remark this answers is not in the document]
Comment (hrjheng) [reply offline/packages/trackreco/PHActsSiliconSeeding.cc:778]: This is intentional
